### ISSUE TITLE
"couldn't sign you in" - this browser or app may not be secure - Kubuntu 20.04 user, using the snap

### ISSUE THREAD

**Issue by ndroftheline** [ytmdesktop/ytmdesktop#981]
Hello, 

I'm running Kubuntu 20.04 with all updates installed, and very little customised. I have the latest/stable snap installed:

youtube-music-desktop-app  1.13.0                      6      latest/stable  adlerluiz   -

Attempting to launch ytmdesktop and log in I'm presented with an error page that says "Couldn't sign you in, This browser or app may not be secure. Learn more, Try using a different browser. If you're already using a supported browser, you can try again to sign in."

Perhaps the underlying browser must be updated, but is there something I can do on my end to get it going? thanks.

**Comment by lisotton** [comment]
I had the same issue this morning both with flatpack and with AppImage.

**Comment by eksortso** [comment]
I don't know if it would work with Kubuntu, but there's a process that you could try that's described in an open issue, here: https://github.com/ytmdesktop/ytmdesktop/issues/866#issuecomment-974205806

TLDR Use Guest Mode (the app's icon will open a fresh window for it) and sign in there; close the app completely; then reopen and sign in.

Does this work for you?
